.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+<!-- Thanks so much for your PR, your contribution is appreciated! -->
+
+- [ ] Description explains the issue / use-case resolved, and auto-closes the related issue(s) (google it).
+- [ ] PR has tests / docs demo, and is linted.
+- [ ] Commit and PR titles begin with [ComponentName], and are in imperative form: "[Component] Fix leaky abstraction".
```

Comment:
Out of curiosity, should we do commit titles with [ComponentName] too? I haven't been following that rule :fearful: